### ISSUE TITLE
Accessibility Issue: Hidden Cookie Banner Content Exposed to Screen Readers (WCAG 1.3.1)

### ISSUE THREAD

**Issue by aesyphers** [rlankhorst/complianz-gdpr#469]
The Complianz cookie banner has elements that are visually hidden (.cmplz-hidden) on the frontend, but not programmatically hidden, causing certain content to be exposed to assistive technologies. For instance, the content below is appearing in Textise when it's not visible on the frontend:

https://www.textise.net/showText.aspx?strURL=https%253A//colinpowellschool.ccny.cuny.edu/#goog_rewarded

**Content that should be hidden using `aria-hidden="true`"**

Functional Functional Always active
The technical storage or access is strictly necessary for the legitimate purpose of enabling the use of a specific service explicitly requested by the subscriber or user, or for the sole purpose of carrying out the transmission of a communication over an electronic communications network.
Preferences Preferences
The technical storage or access is necessary for the legitimate purpose of storing preferences that are not requested by the subscriber or user.
Statistics Statistics
The technical storage or access that is used exclusively for statistical purposes. The technical storage or access that is used exclusively for anonymous statistical purposes. Without a subpoena, voluntary compliance on the part of your Internet Service Provider, or additional records from a third party, information stored or retrieved for this purpose alone cannot usually be used to identify you.
Marketing Marketing
The technical storage or access is required to create user profiles to send advertising, or to track the user on a website or across several websites for similar marketing purposes.

<img width="1000" height="512" alt="Image" src="https://github.com/user-attachments/assets/df6d6058-5ea0-4cfd-bc4f-88067bc82323" />
